yesod-test/Yesod/Test.hs

```diff
@@ -693,6 +694,25 @@ get url = request $ do
     setMethod "GET"
     setUrl url
 
+-- | Follow a redirect, if the last response was a redirect.
+-- ==== __Examples__
+-- > get HomeR
+--
+-- > followRedirect
+followRedirect :: Yesod site
+               =>  YesodExample site (Either T.Text ())
```

Comment:
Would it be more useful to return the URL that was redirected to in the `Right` case? I don't know if that's actually useful, I just originally saw `Either Text ()` and assumed it was `Left <URL redirected to>` and `Right <no redirect>`